test/support/figgy_test_support.ex

```diff
@@ -0,0 +1,13 @@
+defmodule FiggyTestSupport do
```

Comment:
Should this go in test/support/fixtures?

Reply:
The `markers` function is a fixture, but the other two functions are not. Moving the fixtures into a properly namespaced module is a good idea.